my_sol.rb

```diff
@@ -0,0 +1,196 @@
+students = [
+    {
+        first_name: 'John',
+        last_name: 'Doe'
+    },
+    {
+        first_name: 'Jane',
+        last_name: 'Doe',
+    },
+    {
+        first_name: 'Jennifer',
+        last_name: 'Aniston',
+    }
+  ]
+  
+   
+  arr = students.map do |i|
+             i[:first_name].upcase + " " + i[:last_name].upcase
```

Comment:
Great work, but the variable's name represents what information the variable contains so change the variable name from ```i``` to ```student```
```students.each do |student|```